lib/ui/views/home/home_view.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:mobile_app/ui/components/cv_outline_button.dart';
+import 'package:mobile_app/ui/components/cv_subheader.dart';
+import 'package:mobile_app/ui/views/base_view.dart';
+import 'package:mobile_app/ui/views/home/feature_card.dart';
+import 'package:mobile_app/viewmodels/home_viewmodel.dart';
+
+class HomeView extends StatelessWidget {
+  static const String id = "home_view";
+
+  @override
+  Widget build(BuildContext context) {
+    Widget _buildAppBar() {
+      return AppBar(
+        title: Text("CircuitVerse"),
+        centerTitle: true,
+      );
+    }
+
+    Widget _buildHeader() {
+      return Column(
+        children: <Widget>[
+          Text(
+            "Dive into the world of Logic Circuits for free!",
+            style: Theme.of(context).textTheme.headline4.copyWith(
+                  fontWeight: FontWeight.bold,
+                ),
+            textAlign: TextAlign.center,
+          ),
+          Text(
+            "From Simple gates to complex sequential circuits, plot timimg diagrams, automatic circuit generation, explore standard ICs, and much more",
+            style: Theme.of(context).textTheme.subtitle1,
+            textAlign: TextAlign.center,
+          ),
+        ],
+      );
+    }
+
+    Widget _buildHomePageSketch() {
+      return Image.asset(
+        "assets/images/homepage/new-homepage-sketch.png",
+        gaplessPlayback: true,
+      );
+    }
+
+    Widget _buildTeachersAndContributorButtons() {
+      return Container(
+        margin: const EdgeInsets.symmetric(vertical: 32),
+        child: Wrap(
+          alignment: WrapAlignment.center,
+          children: <Widget>[
+            CVOutlineButton(title: "For Teachers"),
+            SizedBox(width: 16),
+            CVOutlineButton(title: "For Contributors"),
+          ],
+        ),
+      );
+    }
+
+    return BaseView<HomeViewModel>(
+      builder: (context, model, child) => Scaffold(
+        appBar: _buildAppBar(),
+        body: SingleChildScrollView(
+          padding: const EdgeInsets.all(16),
+          child: Column(
+            children: <Widget>[
+              _buildHeader(),
+              _buildHomePageSketch(),
+              _buildTeachersAndContributorButtons(),
+              CVSubheader(
+                title: "Features",
+                subtitle:
+                    "Design circuits quickly and easily with a modern and intuitive user interface with drag-and-drop, copy/paste, zoom and more.",
```

Comment:
Can we make this text translatable using i18n ?

Reply:
Yes, so we need to setup internationalisation based on user's device preferences or we must provide them with the options to change language too? @tachyons

Reply:
We don't user preference for the locale yet, so use system locale

Reply:
Sure!